pinot-spi/pom.xml

```diff
@@ -36,6 +36,38 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.7.1</version>
+        <executions>
+          <execution>
+            <id>copy-pinot-plugins</id>
+            <phase>process-test-resources</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
```

Comment:
nit: I think this is a good idea but probably devs are not going to understand on a first read why this plugin is necessary.

Can you add a comment here saying that PluginManagerTest requires these resources and we decided to copy them at build time?

Reply:
Agreed, this could use an extra comment